app/models/forem_instance.rb

```diff
@@ -0,0 +1,9 @@
+class ForemInstance
```

Comment:
A suggestion that came out of #12149 was to add a model (not DB backed) i.e. ForemInstance to which we add a bunch of class methods that encapsulate things that are used for Forem Instances and will change across the different instances. It made sense to start that model with a deployed_at and the latest_commit_id